src/toffy/image_stitching.py

```diff
@@ -150,18 +150,17 @@ def stitch_images(
         )
 
     # get load and stitching args
+    tma_folders = None
     if tiled:
         # returns a dict with keys RnCm and values og folder names
-        folders_dict = get_tiled_names(folders, run_dir)
+        tiled_folders_dict = get_tiled_names(folders, run_dir)
+        tma_folders = ns.natsorted(list(set(folders).difference(set(tiled_folders_dict.values()))))
```

Comment:
This is a long assignment, let's break this up into multiple variables. One for differencing out the tma_folders, one for sorting.